lib/pure/math.nim

```diff
@@ -845,6 +845,18 @@ func round*[T: float32|float64](x: T, places: int): T =
     var mult = pow(10.0, places.T)
     result = round(x*mult)/mult
 
+func `div`*[T: float32|float64](x, y: T): T =
```

Comment:
`T: SomeFloat`

and in future PR, we should also use `SomeFloat` instead of `float32|float64` in this module

Reply:
Does it need since?
`floorDiv` needs this one.

Reply:
This is a recurring problem for added overloads and cases where we add optional parameters. 

The solution is https://github.com/timotheecour/Nim/issues/479 (see `exportSince` + linked PR which allows picking a particular overload symbol) but until then IMO the simplest here is to omit {.since.}. In the unlikely case this cause a regression (or one that's unacceptable) we can either implement https://github.com/timotheecour/Nim/issues/479 or add a one-off `-d:nimLegacyDisableDivFloat`